python/main.py

```
@@ -64,19 +89,34 @@ def hello():
 class AddItemResponse(BaseModel):
     message: str
 
-
 # add_item is a handler to add a new item for POST /items .
 @app.post("/items", response_model=AddItemResponse)
 def add_item(
     name: str = Form(...),
     category: str = Form(...),
-    db: sqlite3.Connection = Depends(get_db),
+    # db: sqlite3.Connection = Depends(get_db),
+    image: UploadFile = None
 ):
     if not name:
         raise HTTPException(status_code=400, detail="name is required")
-
-    insert_item(Item(name=name, category=category))
-    return AddItemResponse(**{"message": f"item received: {name}"})
+    if not category:
+        raise HTTPException(status_code=400, detail="category is required")
+    if image and not image.filename.endswith(".jpg"):
```

[nit] I would recommend not to keep the image field optional, so we can also throw the error if the image itself is None

---

Thank you for the suggestion! I fixed it :)